.gitignore

```diff
@@ -0,0 +1,11 @@
+node_modules/
+dist/
+venv/
+tmp/
+*.log
+.idea
+.vscode/
+.floo
+.flooignore
+package-lock.json
+.DS_Store
```

Comment:
I think we only ignore: nodemodules, dist, package-lock.json.

Reply:
I updated the .gitignore file. Thank you